Platform.XamarinAndroid/Realm.Sync.XamarinAndroid/Realm.Sync.XamarinAndroid.csproj

```diff
@@ -13,7 +13,7 @@
     <AndroidResgenFile>Resources\Resource.Designer.cs</AndroidResgenFile>
     <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
     <AndroidUseLatestPlatformSdk>True</AndroidUseLatestPlatformSdk>
-    <TargetFrameworkVersion>v6.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v7.0</TargetFrameworkVersion>
```

Comment:
Yes - but since it does it all the time, I figured it would be easier to commit and not have to discard it each time :)